exercises/0-intro/src/main.rs

```diff
@@ -0,0 +1,9 @@
+fn main() {
+    let x = 1;
+    let mut y = 1;
+    println!("🦀 Hello, world! 🦀");
+    let x = x + 1;
+    y += 1;
+    println!("You've successfully compiled and run your first Rust project!");
+    println!("X: {x}; Y: {y}");
```

Comment:
During our courses, we've experienced a lot of problems from people that accidentally already had an installation of rust, or sometimes *just* rustc (no cargo). On debian/ubuntu that's unfortunately not uncommon. Usually when they then use rustup to install rust it doesn't do anything (rustc is already there!) but whe they then run code like this it won't compile since you're using in this case the new 1.63 variable formatting. Similarly with bool methods like `then_some` that were added in 1.63 as well. Maybe help people with that.

Reply:
I'll just make this easy for them by providing a `rust-toolchain.toml`. Thanks for the heads up!